src/physics/utilities/solver_config.hpp

```diff
@@ -82,6 +82,12 @@ struct HypreBoomerAMGPrec {
   mfem::ParFiniteElementSpace* pfes = nullptr;
 };
 
+/**
+ * @brief Stores the information required to configure a NVIDIA AMGX preconditioner
+ */
+struct AMGXPrec {
```

Comment:
Why is this just an empty struct?

Reply:
Right now it's only being used as the "tag" in the preconditioner variant, as the default settings appeared to be sufficient for the test I added, but in the future I think we'll want to configure AMGX as we do Hypre's AMG preconditioner implementation.